components/Dashboard/Modals/ProfileDialog/index.js

```diff
@@ -127,6 +129,10 @@ const ProfileDialog = ({ session, isOpen, handleClose, tabs }) => {
     return false;
   };
 
+  const toggleSettings = () => {
+    showSettings ? setShowSettings(false) : setShowSettings(true);
+  };
```

Comment:
toggles are usually written like `setShowSettings(!showSettings)`, which basically means set `showSettings` to the opposite.

Reply:
yes. sorry daddy

Reply:
:sob: